susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -181,6 +181,14 @@ salt-minion-package:
     - makedirs: True
     - require:
       - file: {{ salt_config_dir }}/minion.d/susemanager.conf
+
+{{ salt_config_dir }}/minion.d/beacons.conf:
+  file.managed:
+    - source:
+      - salt://transactional_reboot/reboot_info_beacon.conf
+    - mode: 644
+    - makedirs: True
+
```

Comment:
You probably want to include a `require` here, to ensure this state needs to run after the salt minion package is installed. The execution order of states is not necessary sequential.

I would suggest to add something this here:
```suggestion
    - require:
      - file: {{ salt_config_dir }}/minion.d/susemanager.conf

```